src/views/Viewer.vue

```diff
@@ -1215,8 +1217,16 @@ export default {
 		cursor: pointer;
 	}
 
+	.viewer__content--component {
+		width: 100%;
+		align-self: stretch;
+		display: flex;
+		align-content: center;
+		justify-content: center;
+	}
+
 	&--split {
-		.viewer__file--active {
+		.viewer__content--component {
```

Comment:
Probably need to address #1588 to have clean styles for apps like pdfviewer and richdocuments to work with this.